options.ixx

```diff
@@ -329,6 +333,7 @@ export struct Options
         LOG("\t--dump-read-size=VALUE         \tnumber of sectors to read at once on initial dump, DVD only (default: {})", dump_read_size);
         LOG("\t--overread-leadout             \tdo not limit lead-out to the first hundred sectors, read until drive returns SCSI error");
         LOG("\t--force-unscrambled            \tdo not attempt to read data sectors as audio (BE read method only)");
+        LOG("\t--skip-toc-check               \tdo not check TOC when refining a disc");
```

Comment:
Please rename an option to --force-refine and variable name to force_refine respectively.

Reply:
changed